features/org.mozilla.xulrunner.feature/pom.xml

```diff
@@ -26,6 +26,11 @@
 						</goals>
 					</execution>
 				</executions>
+				<configuration>
+					<excludes>
+						<plugin id="org.mozilla.xulrunner.cocoa.macosx"/>
```

Comment:
why are you adding this exclude ? That could explain why you are not getting osx stuff built.